app/src/componentLibrary/sidebar/sidebar.jsx

```diff
@@ -19,154 +19,97 @@ import PropTypes from 'prop-types';
 import Parser from 'html-react-parser';
 import classNames from 'classnames/bind';
 import { useOnClickOutside } from 'common/hooks';
-import { Navbar } from './navbar';
 import { SidebarButton } from './sidebarButton';
 import styles from './sidebar.scss';
 
 const cx = classNames.bind(styles);
-const HOVER = 'hover';
-const ACTIVE = 'active';
 
 export const Sidebar = ({
-  logoBlockIcon,
+  logoLeftIcon,
+  logoRightIcon,
   createMainBlock,
   topSidebarItems,
-  bottomSidebarItems,
```

Comment:
As we don't have `bottomSidebarItems` anymore, let's rename `topSidebarItems` to `sidebarItems` or just `items`.

Reply:
Done.